tests/builder/builder_test.py

```diff
@@ -2389,6 +2389,22 @@ def test_load_kerning_bracket(ufo_module):
     assert ds2.sources[3].font.kerning == {}
 
 
+def test_unicode_variation_sequences(ufo_module):
+    font = generate_minimal_font()
+    add_glyph(font, "zero")["unicode"] = f"{ord('0'):04x}"
+    add_glyph(font, "zero.uv001")
+    add_glyph(font, "zero.uv255")
```

Comment:
Could we use some standard Unicode variation sequences instead of made up ones for this test?

Reply:
They are all stabdard variation sequences or IVD sequences, except the `zero.uv255` one, which was
supposed to test the last UVS since I couldn’t find any standard one that use
it. There was a typo in the code and the test expectation was wring as well (yay
for TDD), so I’ve added another test and fixed the typo.